compiler/runtime/Helpers.inc

```diff
@@ -368,7 +368,9 @@ SETVAL(RubyHelper_vm_opt_aset_with,       PyHelper_Last+77)
 SETVAL(RubyHelper_vm_setconstant,         PyHelper_Last+78)
 SETVAL(RubyHelper_rb_vm_env_write,        PyHelper_Last+79)
 SETVAL(RubyHelper_vm_jit_stack_check,     PyHelper_Last+80)
-SETVAL(TR_FSRH,                           PyHelper_Last+81)
+SETVAL(RubyHelper_rb_str_freeze,          PyHelper_Last+81)
+SETVAL(RubyHelper_rb_ivar_set,            PyHelper_Last+82)
+SETVAL(TR_FSRH,                           PyHelper_Last+82)
```

Comment:
Should this be `PyHelper_Last+83`?  The next `SETVAL` operates on `TR_FSRH+0` which overlaps with `rb_ivar_set`.